packages/local-environment/tests/nomadenv.test.ts

```diff
@@ -0,0 +1,47 @@
+import { HardhatNetwork } from "../src/network";
+import { NomadDomain } from "../src/domain";
+import { expect, assert } from "chai";
+import { NomadEnv } from "../src/nomadenv";
+
+describe("NomadDomain test", () => {
+    //TODO: We should implement any-network connection logic and test accordingly.
+    it('can create a valid NomadEnvironment', async () => {
+        // Creation
+        const t = new HardhatNetwork('tom', 1);
+        const j = new HardhatNetwork('jerry', 2);
+        const tDomain = new NomadDomain(t);
+        const jDomain = new NomadDomain(j);
+        const le = new NomadEnv({
+            domain: tDomain.network.domainNumber,
+            id: "0x" + "20".repeat(20),
+        });
+        expect(le).to.exist;
+        expect(le.govNetwork).to.equal(tDomain);
+        // SDK
+        expect(le.bridgeSDK).to.exist;
+        expect(le.coreSDK).to.exist;
+
+        // Can add domains
+        le.addDomain(tDomain);
+        le.addDomain(jDomain);
+        assert.isTrue(le.domains.includes(tDomain));
+        assert.isTrue(le.domains.includes(jDomain));
+
+        // Can up agents
```

Comment:
Here I might go so far as to check if docker containers are running as opposed to checking `isAgentUp` alone

Reply:
Implemented!

Reply:
@yourbuddyconner Thoughts on running tests in order? Tests are now running one by one but I'm not exactly sure as to how to specify the order in which the tests run.

Reply:
I don't think order matters, especially if tests clean up after themselves